### DISCUSSION TITLE
JSON's in Flexible Content

### DISCUSSION THREAD

**Discussion by Kaylakaze** [whitecube/nova-flexible-content#301] [Q&A]
I've tried the two JSON wrapper packages I could find as well as trying manually with "x->y" syntax and in no case can I get a Flexible content field to return the data in a {"x": {"y": "blah"}} sort of way, seemingly due to all the added formatting Flexible Content adds to the JSON array. I don't need multiple different layouts, and I'd be willing to bet that most people don't most of the time. It'd be nice if a flexible field only has one layout, if it could just return a freaking normal JSON instead of one with all its extra junk in it. Not only should that play well with nested JSONs but would also keep me from having to make resolvers to convert my data from a normal JSON, as it needs to be in the database, to the layout format.

Does anyone have any ideas how to get JSONs to work in flexible fields, either using the "x->y" syntax or using a wrapper like 64robot's?

**Comment by muni-oscar-parra** [comment]
x2
